tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Cohen's Kappa"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import tensorflow as tf
+import tensorflow.keras.backend as K
+from tensorflow.math import confusion_matrix
+from tensorflow.keras.metrics import Metric
```

Comment:
I think we could directly use `tf.math.confusion_matrix` and `tf.keras.metrics.Metric` though it is not a private API...

Reply:
We can but either way should be fine, right?

Reply:
Sure, I'm also fine with this.